scripts/start.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+# This script starts the application.
+# Based on the environment it may also deploy the smart contracts.
+
+if [ -n "$CI" ]; then
+  # Before running the application, when in CI, we first need to deploy the smart contracts
+  # The artifact needs to be re-written before the application starts
+  cd smart-contracts
```

Comment:
I think we can leverage  bash subshell here and down below. https://www.tldp.org/LDP/abs/html/subshells.html